main.py

```diff
@@ -1,10 +1,43 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_info = json.load(players_file)
+
+    for player, player_info in players_info.items():
+        guild = player_info["guild"] if player_info["guild"] else None
+        if guild:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild["name"],
+                description=guild["description"]
+            )
+
+        race = player_info["race"]
+        Race.objects.get_or_create(
+            name=race["name"],
+            description=race["description"]
+        )
+        racing = Race.objects.get(name=race["name"])
```

Comment:
could u create this variable using get_or_create method before ?) why do u need to make query to db twice ?